dev/com.ibm.ws.microprofile.reactive.messaging_fat/fat/src/com/ibm/ws/microprofile/reactive/messaging/fat/kafka/mtls/mutliple/singleapp/KafkaMtlsMultipleChannelsTest.java

```diff
@@ -0,0 +1,130 @@
+
+        // Channel Properties should take priority over Connector Properties. So provide valid keystore to channels and invalid to Connector.
+        // If invalid keystore is used then the SSL Handshake will fail
+        ConnectorProperties outgoingProperties2 = simpleOutgoingChannel(null, MessagingBeanTwo.CHANNEL_OUT)
+                // Valid Keystore
+                .addProperty(SslConfigs.SSL_KEYSTORE_LOCATION_CONFIG, KafkaUtils.KEYSTORE2_FILENAME)
+                .addProperty(SslConfigs.SSL_KEYSTORE_PASSWORD_CONFIG, MtlsMultipleKeyStoresTests.kafkaContainer.getKeystorePassword());
+
+        ConnectorProperties incomingProperties2 = simpleIncomingChannel(null, MessagingBeanTwo.CHANNEL_IN, APP_GROUP_ID2)
+                // Valid Keystore
+                .addProperty(SslConfigs.SSL_KEYSTORE_LOCATION_CONFIG, KafkaUtils.KEYSTORE2_FILENAME)
+                .addProperty(SslConfigs.SSL_KEYSTORE_PASSWORD_CONFIG, MtlsMultipleKeyStoresTests.kafkaContainer.getKeystorePassword());
+
+        ConnectorProperties connectorProperties = new ConnectorProperties(ConnectorProperties.Direction.CONNECTOR, "liberty-kafka")
+                .addAll(MtlsMultipleKeyStoresTests.testConnectionProperties());
+
+        // The Connection properties for the KafkaTestClient should include all certificates, so it uses the truststore as both trust and key stores so it has both client certs
+        PropertiesAsset appConfig = new PropertiesAsset()
+                .addProperty(KafkaTestClientProvider.CONNECTION_PROPERTIES_KEY, KafkaTestClientProvider.encodeProperties(MtlsMultipleKeyStoresTests.connectionProperties()))
+                .include(incomingProperties)
+                .include(outgoingProperties)
+                .include(incomingProperties2)
+                .include(outgoingProperties2)
+                .include(connectorProperties);
+
+        WebArchive war = ShrinkWrap.create(WebArchive.class, APP_NAME + ".war")
+                .addAsLibraries(kafkaClientLibs())
+                .addAsManifestResource(kafkaPermissions(), "permissions.xml")
+                .addPackage(KafkaMultipleMtlsTestServlet.class.getPackage())
+                .addPackage(MessagingBeanOne.class.getPackage())
+                .addPackage(AbstractKafkaTestServlet.class.getPackage())
+                .addPackage(KafkaTestConstants.class.getPackage())
+                .addAsResource(appConfig, "META-INF/microprofile-config.properties");
+
+        ShrinkHelper.exportDropinAppToServer(server, war, SERVER_ONLY);
+
+        KafkaUtils.copyTrustStore(MtlsMultipleKeyStoresTests.kafkaContainer, server);
+        KafkaUtils.copyKeyStoresToServer(MtlsMultipleKeyStoresTests.kafkaContainer, server);
+
+        server.startServer();
+    }
+
+    @Test
+    public void testMutlipleMtlsChannels() throws Exception {
+        runTest(server, APP_NAME + "/KafkaMultipleMtlsTestServlet", "testMtls");
+    }
```

Comment:
Use `@Test` on `KafkaMultipleMtlsTestServlet.testMtls` and `@TestServlet` on the `LibertyServer` field.